README.md

```diff
@@ -143,14 +149,17 @@ await analytics.setup('writeKey', {
   using: [firebase]
 })
 ```
-For IOS you will need to add the GoogleService-info.plist to your IOS folder. This file can be downloaded from your Firebase instance. Firebase takes up to 24 hours to show events. However, you can utilize the Firebase debug method to confirm your setup is sending data correctly. To do this add ```-FIRDebugEnabled``` in Xcode’s Scheme Settings. 
 
-In Xcode: 
-```Project -> Scheme -> Edit Scheme -> Arguments Passed On Launch```
+For IOS you must add the GoogleService-info.plist to your iOS folder. This file can be downloaded from your Firebase instance. Firebase takes up to 24 hours to show events. However, you can utilize the Firebase debug method to confirm your setup is sending data correctly. To do this add `-FIRDebugEnabled` in Xcode’s Scheme Settings.
+
+In Xcode:
+`Project -> Scheme -> Edit Scheme -> Arguments Passed On Launch`
+
+#### Supported Device-Mode Destinations
 
-#### Integrations
+> All destinations have the same version as `@segment/analytics-react-native`
 
-> All integrations have the same version as `@segment/analytics-react-native`
+**Note**: Each device-mode destination has a different native setup procedure due to differences between the underlying SDK vendors. Please refer to the vendor documentation for configuring the native iOS and Android portions of a given destination. More information and links to vendor specific instructions and details can be found at [Connection Mode Comparisons](https://segment.com/docs/connections/destinations/cmodes-compare/).
```

Comment:
```suggestion
**Note**: Each device-mode destination has its own set up steps because of differences between the underlying SDK vendors. Check the vendor's documentation for instructions on how to configure the native iOS and Android parts of each destination. More information, and links to vendor specific instructions and details, can be found at [Connection Mode Comparisons](https://segment.com/docs/connections/destinations/cmodes-compare/).
```